<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataRecordUI</class>
 <widget class="QWidget" name="dataRecordUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System - DataRecord</string>
  </property>
  <widget class="QGroupBox" name="faceRecordGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>人脸采集</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="faceDetectCaptureLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>&lt;font color=red&gt;摄像头未开启&lt;/font&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>641</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="faceRecordHorizontalLayout">
     <item>
      <widget class="QPushButton" name="startFaceRecordButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>开始采集人脸数据</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="faceRecordCountHorizontalLayout">
       <item>
        <widget class="QLabel" name="faceRecordCountLabel">
         <property name="text">
          <string>系统已采集帧数：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="faceRecordCountLcdNum">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="enableFaceRecordButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>采集当前捕获帧</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>21</y>
      <width>641</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="controlHorizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="webcamHorizontalLayout">
       <item>
        <widget class="QCheckBox" name="useExternalCameraCheckBox">
         <property name="text">
          <string>使用外接摄像头</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startWebcamButton">
         <property name="text">
          <string>打开摄像头</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="enableFaceDetectButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>开启人脸检测</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget_2</zorder>
   <zorder>faceDetectCaptureLabel</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QGroupBox" name="dataGroupBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>341</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>数据录入</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="userInfoGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>基本信息</string>
    </property>
    <widget class="QGroupBox" name="infoConfirmGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>301</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>信息确认</string>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="infoConfirmHorizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="keyVerticalLayout">
         <item>
          <widget class="QLabel" name="stuIDLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>工号：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cnNameLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>姓名：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="enNameLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>汉语拼音：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="valueVerticalLayout">
         <item>
          <widget class="QLineEdit" name="stuIDLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cnNameLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enNameLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="migrateToDbButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>同步到数据库</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="addOrUpdateUserInfoButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>23</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>增加用户/修改用户资料</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="logGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>系统日志</string>
    </property>
    <widget class="QTextEdit" name="logTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>321</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dbGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>数据库状态</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="dbUserCountHorizontalLayout">
      <item>
       <widget class="QLabel" name="dbUserCountLabel">
        <property name="text">
         <string>数据库已存人脸样本数：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="dbUserCountLcdNum">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="initDbButton">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化数据库</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
